cmd/install/install.go

```diff
@@ -69,7 +71,8 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.ExcludeEtcdManifests, "exclude-etcd", false, "Leave out etcd manifests")
 	cmd.Flags().BoolVar(&opts.EnableOCPClusterMonitoring, "enable-ocp-cluster-monitoring", opts.EnableOCPClusterMonitoring, "Development-only option that will make your OCP cluster unsupported: If the cluster Prometheus should be configured to scrape metrics")
 	cmd.Flags().BoolVar(&opts.EnableCIDebugOutput, "enable-ci-debug-output", opts.EnableCIDebugOutput, "If extra CI debug output should be enabled")
-	cmd.Flags().StringVar(&opts.AWSCreds, "aws-creds", opts.AWSCreds, "Path to an AWS credentials file (AWS only)")
+	cmd.Flags().StringVar(&opts.AWSCreds, "aws-creds", opts.AWSCreds, "Path to an AWS credentials file")
+	cmd.Flags().StringVar(&opts.Region, "region", opts.Region, "AWS region in which the operator will run")
```

Comment:
Should we have one region flag for everything or rather have a "per thing" region flag (oidc bucket, service endpoints)?

Reply:
Yes, we are each trying to add this in pending PRs.  I prefer just `--region` vs `--oidc-bucket-region` so we can use it generally.

Reply:
ok, but lets make it at least `aws-region` so we don't conflict once we need another provides region

Reply:
what about a `--platform` and `--region` flag?

Reply:
that implies that a hypershift instance will only manage guest clusters for one provider, are we sure this will hold true?